README.md

```diff
@@ -8,32 +8,35 @@ Edit your OpenApi specification with your favorite IDE and test it with swagger-
 
 ## Installation
 
-Describe how to install the project/library/framework/…
+```bash
+docker build . -t swagger-editor-lepetitbloc
```

Comment:
This command would be more appropriate for development.
For an installation it should be as simple as `docker run lepetitbloc/swagger-editor`